touchforms/formplayer/experiments.py

```diff
@@ -136,15 +137,18 @@ def formplayer_string_compare(control, candidate, key=None):
                      % (key, control, candidate))
     return ret
 
-## Keys that SMS doesn't use or that we otherwise can skip
-def check_skip_key(key):
+## Keys that we can't test the content of, but want to be present in the candidate
+def check_skip_content(key):
     if key == "session_id":
         # clearly these will be different
         return True
     elif key == "output":
         # don't have any way to compare the XML output at the moment, esp. considering uuids and times
         return True
-    elif key == "seq_id":
```

Comment:
would be good to explicity return False here (it will currently return None which will still work for  this use case but better to be explicit)